### ISSUE TITLE
Discussion point for later, disagreement between censored, dead, status and fate

### ISSUE THREAD

**Issue by Huh** [Huh/PopR_SDGFP#57]
Some animals have some disagreement between the Status, Fate, Censored and Dead columns.  

One thing we could do in the code is to look for n pieces of evidence for censored and dead and then assign the state with the most evidence.  Alternatively, we can delete these animals and report the loss of the individuals, which seems safer and brings data issues to the forefront, which is important.

See the last line of individual 20090702's encounter history as an example of what I am describing above.

```r
      ID           Species BioYear Month   DAU  Unit Ageclass    Sex       Date Status  Fate Censored  Dead MortSigHeard  CollectionType Latitude Longitude cap_age num_age Region
   (dbl)             (chr)   (dbl) (dbl) (chr) (chr)    (chr)  (chr)     (date)  (dbl) (chr)    (lgl) (lgl)        (chr)           (chr)    (dbl)     (dbl)   (dbl)   (dbl)  (int)
1      1 White-tailed Deer    2012     1    NA   12A    Adult Female 2011-10-01      1    NA    FALSE FALSE           NA Deer Monitoring       NA        NA       3       3      3
2      1 White-tailed Deer    2012     2    NA   12A    Adult Female 2011-11-01      1    NA    FALSE FALSE           NA Deer Monitoring       NA        NA       3       3      3
3      1 White-tailed Deer    2012     3    NA   12A    Adult Female 2011-12-01      1    NA    FALSE FALSE           NA Deer Monitoring       NA        NA       3       3      3
4      1 White-tailed Deer    2012     4    NA   12A    Adult Female 2012-01-01      1    NA    FALSE FALSE           NA Deer Monitoring       NA        NA       3       3      3
5      1 White-tailed Deer    2012     5    NA   12A    Adult Female 2012-02-01      1    NA    FALSE FALSE           NA Deer Monitoring       NA        NA       3       3      3
6      1 White-tailed Deer    2012     6    NA   12A    Adult Female 2012-03-01      1    NA    FALSE FALSE           NA Deer Monitoring       NA        NA       3       3      3
7      1 White-tailed Deer    2012     7    NA   12A    Adult Female 2012-04-01      1    NA    FALSE FALSE           NA Deer Monitoring       NA        NA       3       3      3
8      1 White-tailed Deer    2012     8    NA   12A    Adult Female 2012-05-01      3 other     TRUE FALSE           NA  Deer Mortality       NA        NA       3       3      3
```

**Comment by Huh** [comment]
To expand just a bit, in PopR try settings:
Species = White-tailed Deer
Region = 3
Year = 2012
Sex = Female
Age = Adult

Then look at the raw data and almost every individual dies by other cause in May of 2012.  This does not seem likely.

**Comment by gfrc11752** [comment]
After reviewing the above data set, May of 2012 was the last date those animals were monitored thus all being censored during that month. It appears to me 17 individuals were censored in May of 2012 and were not entered as dead. Dead = FALSE and Censored = True

**Comment by Huh** [comment]
Agreed, but status equals 3 (death by other).  That was true for my example animal not so sure about the others.


Josh NowakUniversity of Montana

-------- Original message --------

**Comment by gfrc11752** [comment]
Yeah, they should be a 1 (alive); however, if censored=true it shouldn’t matter if the animal is declared dead or is still alive because we are censoring it right?

**Comment by Huh** [comment]
I guess it depends on which column you believe the most.  Many animals have a T in the Censored column when they are captured (likely my fault).  
My concern is that I don't know what to think when the animal is recorded as having state 3, fate other, dead false and censored true.  If I just fill in dead or censored, give one priority, then we have potential to seriously influence the results of the analyses. Josh Nowak 


     On Tuesday, October 6, 2015 10:19 AM, gfrc11752 <notifications@github.com> wrote:
   

 Yeah, they should be a 1 (alive); however, if censored=true it shouldn’t matter if the animal is declared dead or is still alive because we are censoring it right?

**Comment by gfrc11752** [comment]
With the exception of the Capture record saying censored=True in some cases; I believe the Censored=true column is the most reliable regardless of fate. We could write a line that would change fate to censored if dead = false, censored = true.

**Comment by Huh** [comment]
This behavior was changed to match the suggestion to rely most heavily on the Censored column.  However, the data still look the same in terms of having an entry for Fate and being Censored.

In PopR you can set species to White-tailed Deer and then click on raw data and enter 2009070 in the ID column filter.  The second and third rows display the behavior of the data handling.

I am of the opinion that Fate only relates to a form of death and other should be reserved for forms of death.  A description of censor type would be reasonable here for censored animals.
